addons/note/views/note_views.xml

```diff
@@ -122,7 +122,7 @@
                         <img
                             t-if="follower_index &lt; 5"
                             t-att-src="kanban_image('res.partner', 'avatar_128', follower)"
-                            class="oe_kanban_avatar o_image_24_cover rounded-circle border border-white bg-white ml-n2"
```

Comment:
this should be a fixup of https://github.com/odoo/odoo/pull/116641/commits/5cee5a3de562b912fa1d10b25a3b9e48845dd050